packages/Webkul/Admin/src/Resources/views/components/datagrid/index.blade.php

```diff
@@ -333,7 +333,9 @@
                     /**
                      * We need to reset the page on filtering.
                      */
-                    this.applied.pagination.page = 1;
+                    if (! appliedColumn.value.length) {
+                        this.applied.filters.columns = this.applied.filters.columns.filter(column => column.index !== columnIndex);
+                    }
```

Comment:
No need to change this. It should be 
this.applied.pagination.page = 1;

Reply:
Done